src/main/java/codeu/model/data/Activity.java

```diff
@@ -0,0 +1,76 @@
+package codeu.model.data;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.Locale;
+import java.util.UUID;
+import java.util.List;
+
+
+public class Activity implements Comparable<Activity> {
+	public final UUID id;
+	public final Instant creation;
+	public final String type;
+	public final List<String> attributes;
+
+	public Activity(UUID id, Instant creation, String type, List<String> attributes) {
+		this.id = id;
+		this.creation = creation;
+		this.type = type;
+		this.attributes = attributes;
+		
+	}
+	
+	public UUID getId(){
+		return this.id;
+	}
+	
+	public Instant getCreationTime(){
+		return this.creation; 
+	}
+	
+	public String getType(){
+		return this.type;
+	}
+	
+	public List<String> getAttributes(){
+		return this.attributes;
+	}
+	
+	public String getStringAttributes() {
+		return String.join(",", this.attributes);
+	}
+	
+	
+	public String toString(){
+		if (this.type.equals("conversation")) {
```

Comment:
A more OOP (objected oriented programming) way to do this is to create three subtypes of Activity and then have them all override the toString method differently.